content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/starshipit_shipping.rst

```diff
@@ -0,0 +1,201 @@
+----------------------------------
+
+In your Starshipit account, navigate to :menuselection:`Settings --> API` in the side menu.
+There, you can find the API keys you will need to connect to Odoo.
+
+.. image:: starshipit_shipping/starshipit-settings-api.png
+   :align: center
+   :alt: Finding the Starshipit API keys.
+
+Setup in Odoo
+=============
+
+Install the Starshipit shipping module
+--------------------------------------
+
+After the Starshipit account is set up and configured, it's time to configure the Odoo database.
+To get started, go to Odoo's :guilabel:`Apps` module,  search for the :guilabel:`Starshipit
+Shipping` integration, and install it.
+
+.. image:: starshipit_shipping/starshipit-app.png
+   :align: center
+   :alt: Starshipit Shipping module in the Odoo Apps module.
+
+Starshipit shipping connector configuration
+-------------------------------------------
+
+Once installed, activate the :guilabel:`Starshipit Shipping` module in :menuselection:`Inventory -->
+Configuration --> Settings`. The :guilabel:`Starshipit Connector` setting is found under the
+:guilabel:`Shipping Connectors` section.
+
+After activating the :guilabel:`Starshipit Connector`, click on the :guilabel:`Starshipit Shipping
+Methods` link below the listed connector. Once on the :guilabel:`Shipping Methods` page, click
+:guilabel:`Create`.
+
+.. tip::
+   :guilabel:`Shipping Methods` can also be accessed by going to :menuselection:`Inventory -->
+   Configuration --> Delivery --> Shipping Methods`.
+
+Fill out the following fields in the :guilabel:`New Shipping Method` form:
+
+- :guilabel:`Shipping Method`: type `Starshipit`.
+- :guilabel:`Provider`: select :guilabel:`Starshipit` from the drop-down menu.
+- :guilabel:`Delivery Product`: set the product that was configured for this shipping method or
+  create a new product.
+- In the :guilabel:`Starshipit Configuration` tab, enter the :guilabel:`Default Package Type`
+  if you are going to use packages during shipping in order to get more accurate rates.
+  The packages system may be enabled in the Odoo settings.
+- In the :guilabel:`Starshipit Configuration` tab, enter the :guilabel:`Origin Address`.
+  It will be used as sending address when getting available services for your integration.
+- In the :guilabel:`Starshipit Configuration` tab, enter the :guilabel:`Starshipit API Key`.
+- In the :guilabel:`Starshipit Configuration` tab, enter the :guilabel:`Starshipit Subscription Key`
+  .
+- Manually :guilabel:`Save` the form by clicking the cloud icon next to the :guilabel:`Shipping
+  Methods / New` breadcrumbs.
+
+After configuring and saving the form, follow these steps to load the shipping products:
+
+- In the :guilabel:`Starshipit Configuration` tab of the :guilabel:`New Shipping Method` form, click
+  on the :guilabel:`Select a service linked to your Starshipit account` link.
+- Select the shipping service to be used for deliveries and returns.
+- Click :guilabel:`Confirm`.
```

Comment:
Totally personal preference, but we can go into more detail describing the pop-up window
```suggestion
To load the newly configured shipping products, click the :guilabel:`Select a service linked to your
Starshipit account` link at the bottom of the :guilabel:`Starshipit Configuration` tab.

Doing so opens the :guilabel:`Choose Starshipit Shipping Service` pop-up window. In the
:guilabel:`Delivery Service` field, choose the desired shipping service for deliveries and returns
from the drop-down menu. Finally, click :guilabel:`Confirm`.

The chosen delivery service will populate in the :guilabel:`Service Name` field.

```